bin/node/runtime/pangolin/src/evm.rs

```diff
@@ -8,12 +8,12 @@ use darwinia_evm::{
 use darwinia_evm_precompile::DarwiniaPrecompiles;
 use dvm_ethereum::account_basic::DVMAccountBasicMapping;
 
-/// Fixed gas price of `1`.
+/// Fixed gas price.
 pub struct FixedGasPrice;
 impl FeeCalculator for FixedGasPrice {
 	fn min_gas_price() -> U256 {
 		// Gas price is always one token per gas.
-		1.into()
+		100_000_000_000u128.into()
```

Comment:
About this value, I would like to hear your advice. @HackFisher

Reply:
@AsceticBear What's the meaning of minimal gas price, is this some fixed gas price which can not be changed by user?

What will happen if user manually change it to lower gas price, e.g. 1 Gwei, in Metamask?